src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php

```diff
@@ -198,11 +201,7 @@ public function handle(AddProductToOrderCommand $command)
 
             // Update totals amount of order
             $this->orderAmountUpdater->update($order, $cart, (int) $orderDetail->id_order_invoice);
```

Comment:
I believe we made a mistake here, it would be more logic to call the hook after the update (like in the updater handler) or the module won't receive the correct total in the order

Reply:
In theory we should not follow logic but rather attempt to mimic previous position of hook. So even if previous hook dispatch was not logically placed we must try to be as close as possible to it 😄  else modules behavior will be impacted